pkg/v1/tkg/tkgctl/utils.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package tkgctl
+
+import "github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/constants"
+
+func (t *tkgctl) setCEIPOptinBasedOnConfigAndBuildEdition(edition string) string {
+	ceipOptIn, err := t.TKGConfigReaderWriter().Get(constants.ConfigVariableEnableCEIPParticipation)
+	if err == nil {
+		return ceipOptIn
+	} else if edition == TCEBuildEdition {
+		return False
+	} else {
+		return True
+	}
```

Comment:
nit: We can reduce the if... else if ... else and convert it into 4 if statements here as we have `return` in each if statement.

Reply:
Thanks! Updated.